Maven build failing; improper SDK level in pom.xml

`mvn clean verify` will fail on a fresh checkout with the following error:

```
[ERROR] Failed to execute goal com.jayway.maven.plugins.android.generation2:android-maven-plugin:3.4.1:generate-sources (default-generate-sources) on project android-times-square: MojoExecutionException: ANDROID-040-001: Could not execute: Command = /bin/sh -c cd /Users/tad/git/android-times-square/library && /Users/tad/dev/sdk/platform-tools/aapt package -m -J /Users/tad/git/android-times-square/library/target/generated-sources/r -M /Users/tad/git/android-times-square/library/AndroidManifest.xml -S /Users/tad/git/android-times-square/library/res --auto-add-overlay -A /Users/tad/git/android-times-square/library/assets -I /Users/tad/dev/sdk/platforms/android-16/android.jar, Result = 1 -> [Help 1]
```

The output of that command is:

```
/Users/tad/git/android-times-square/library/res/values/styles.xml:13: error: Error: No resource found that matches the given name: attr 'android:textAlignment'.
```

The `textAlignment` resource ID was added in API 17. Changing the `android.platform` value to 17 in `pom.xml` solves the problem.

--- a/pom.xml
+++ b/pom.xml
@@ -49,7 +49,7 @@
 
     <java.version>1.6</java.version>
     <android.version>4.1.1.4</android.version>
-    <android.platform>16</android.platform>
+    <android.platform>17</android.platform>
 
     <fest-assert.version>2.0M8</fest-assert.version>
     <junit.version>4.10</junit.version>

